extensions/vscode/src/views/homeView.ts

```diff
@@ -979,6 +984,53 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
     return commands.executeCommand(Commands.Logs.Focus);
   }
 
+  private async showAssociateGUID() {
+    let urlOrGuid = "";
+    const result = await window.showInputBox({
+      title: "Enter the URL of the Existing Content Item on the Server.",
+      prompt:
+        "You can supply either the content's dashboard URL or the solo URL.",
+      value: urlOrGuid,
+      placeHolder:
+        "For example: https://connect.company.co/connect/#/apps/adffa505-08c7-450f-88d0-f42957f56eff",
+      validateInput: (text) => {
+        const guid = extractGUID(text);
+        if (guid === null) {
+          return Promise.resolve({
+            message: `Unexpected format for a Posit Connect Content URL. Confirm URL value loads content from server and contains a content GUID.`,
```

Comment:
Wording is tough, and I quite liked this error 👌 

```suggestion
            message: `Unexpected format for a Posit Connect Content URL. Confirm the URL loads content from the server and contains a content GUID.`,
```

Reply:
updated!